modules/pel/peltool/peltool.py

```diff
@@ -327,6 +412,17 @@ def main():
 
         sys.exit(0)
 
+    if args.list:
+        if inBMC:
+            listOption(BMCPELsPath, config, args.extension, args.delete)
+            sys.exit(0)
+        else:
+            if not os.path.isdir(args.list):
+                print("Error: Given path is not a directory")
```

Comment:
sorry, one minor thing:  to be consistent, lets exit how it was done on line 307 and use the sys.exit() option that takes a string.

also, you can move the sys.exit(0) to the bottom of the if args.list block and then just have it once